setup.py

```diff
@@ -864,12 +866,39 @@ def build_extensions(self):
             except:
                 pass
 
+        build_ucx = int(os.environ.get('BYTEPS_WITH_UCX', 1))
+        print("build_ucx is", build_ucx)
+        if build_ucx:
+            ucx_path = pre_setup.ucx_path.strip()
+            if not ucx_path:
+                ucx_path = "https://codeload.github.com/openucx/ucx/zip/9229f54"
```

Comment:
Is there a better way to make the version of UCX configurable?

Reply:
L874 is a hard-coded commit in the ucx master branch, it can be overridden in pre_setup.py. We can also supply the default path using an environment variable. Any suggestions?

Reply:
Using an env var looks good to me.